project.godot

```diff
@@ -8,13 +8,18 @@
 
 config_version=5
 
+[addons]
+
+git_describe/commit_hash="e374f8b6e94f36948a2016c172b57fa68737c4db"
+
 [application]
 
 config/name="Threadbare"
 config/tags=PackedStringArray("2d", "endless", "game", "tilemap", "top_down")
 run/main_scene="uid://huuo8mnwsphv"
 config/features=PackedStringArray("4.4", "GL Compatibility")
 config/icon="uid://bc70phmq55dkf"
+config/git_describe="e374f8b"
```

Comment:
These changes are due committing while the game is running – I filed #1163 about a way we could avoid this happening. For now, quit Godot, then use the following command to revert these changes:

```bash
git restore --source=HEAD~ project.godot
git add project.godot
git commit -m 'Revert unintended changes to project.godot'
```

Reply:
`HEAD~` means "the commit before the current one".